esmvalcore/_data_finder.py

```diff
@@ -253,41 +290,42 @@ def get_input_filelist(variable, rootpath, drs):
     if variable['project'] == 'CMIP5' and variable['frequency'] == 'fx':
```

Comment:
```suggestion
    if all([variable['project'] == 'CMIP5',
            variable['frequency'] == 'fx',
            variable.get('ensemble') == '*']):
```

Reply:
I'm not sure of the intention here. Is it to not override the ensemble number for CMIP5 when the user specifies ``*`` for the ensemble? In that case is there a typo here and you mean ``!=`` in the last comparison?

Reply:
Yes, I mean `!=`. Sorry!!